Simplified/NYPLWelcomeScreen.swift

```diff
@@ -284,7 +284,7 @@ import PureLayout
 final class NYPLWelcomeScreenAccountList: UIViewController, UITableViewDelegate, UITableViewDataSource {
   
   var accounts: [Account]!
-  var nyplAccounts: [Account]!
+  var nyplUserAccounts: [Account]!
```

Comment:
Also not NYPLUserAccount objects in this array

Reply:
fixed